lib/command_definitions.ts

```diff
@@ -146,3 +146,7 @@ export let rotationGesture = new CommandDefinition(
     });
 export let shakeDevice =
     new CommandDefinition<void>('shakeDevice', [], 'POST', 'appium/device/shake');
+export let sendCommand =
+    new CommandDefinition<void>('sendCommand', ['cmd', 'params'], 'POST', '/chromium/send_command');
+export let sendCommandAndGetResult = new CommandDefinition<Object>(
+  'sendCommandAndGetResult', ['cmd', 'params'], 'POST', '/chromium/send_command_and_get_result');
```

Comment:
Just curious — since I am unfamiliar with these chromium commands — is there a practical difference between them besides not returning a result?

Reply:
Not really. I'm just exposing the two commands available in the devtools client. Not all devtools commands will necessarily return a result.